app/src/main/java/edu/uc/kansakpb/ikeahjalpare3/FurnitureActivity.kt

```diff
@@ -1,11 +1,14 @@
 package edu.uc.kansakpb.ikeahjalpare3
 
+import android.net.Uri
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import kotlinx.android.synthetic.main.activity_furniture.*
 
 class FurnitureActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_furniture)
+        /** TODO: load the URL of the instruction manual in a webView */
```

Comment:
A couple of thoughts here:
1) This feels like conversation... which belongs in the pull request, not source code comments.
2) TODOs are usually in // , not in /** */